src/sentry/integrations/discord/client.py

```diff
@@ -62,54 +57,19 @@ def get_guild_name(self, guild_id: str) -> str:
         response = self.get(GUILD_URL.format(guild_id=guild_id), headers=self.prepare_auth_header())
         return response["name"]  # type: ignore
 
-
-class DiscordClient(IntegrationProxyClient):
```

Comment:
Removing the integration proxy will help with reliability and operations, but comes with an increased cognitive cost of remembering which integrations need to use the proxy and which don't and why.  Keeping that in order as we add more integrations could be challenging for future us 🤔